include/kunit

```diff
@@ -93,6 +93,7 @@ need_kconfig:
 - FAT_KUNIT_TEST: m
 
 # below are kunit test configs of bool type
+- MEMCPY_SLOW_KUNIT_TEST: y
```

Comment:
there's already code to disable this

# disable slow memcpy kunit test which are also part of MEMCPY_KUNIT_TEST but exhaustive
- MEMCPY_SLOW_KUNIT_TEST: n

Reply:
ok